fieldmanager.php

```diff
@@ -18,7 +18,7 @@
 /**
  * Current version of Fieldmanager.
  */
-define( 'FM_VERSION', '1.3.0' );
+define( 'FM_VERSION', '1.3.1' );
```

Comment:
The `main` branch has had several changes merged since the release of `1.3.0`, so I don't think it'd be accurate to label it `1.3.1` after this is merged. I would suggest leaving this line at `1.3.0` for the purpose of this particular PR. 

If you think the fix should also be backported to create a new 1.3.1 release, perhaps we can do that by creating a `1.3` branch based on the `1.3.0` tag (https://github.com/alleyinteractive/wordpress-fieldmanager/releases/tag/1.3.0) and merging this change into that branch in a separate PR, also bumping the version.

Reply:
Good point :) Removed the version update